example/libclang-example/generated_bindings.dart

```diff
@@ -10990,7 +11142,6 @@ final class IndexerCallbacks extends ffi.Struct {
               CXIdxClientContainer Function(
                   CXClientData client_data, ffi.Pointer<ffi.Void> reserved)>>
       startedTranslationUnit;
-
   external ffi.Pointer<
```

Comment:
I don't know when this change happened, but it's not caused by this PR. I'm just the first one to regenerate this file in a while. We didn't notice the change because our unit tests were intentionally ignoring all the empty lines when diffing the expected and actual output. I've removed this behavior so we'll notice such changes in future.